### PR TITLE
move weekly digest email to sidekiq-cron and remove whenever

### LINKED ISSUE #373: Move whenever cron job to sidekiq-cron
https://github.com/coopdevs/timeoverflow/blob/develop/Gemfile#L20
https://github.com/coopdevs/timeoverflow/blob/develop/config/schedule.rb

https://trello.com/c/vCkaG7TW/124-move-whenever-cron-job-to-sidekiq-cron

### LINKED ISSUE #544: App prints in english on test environment and it crushes
We've found out that the app prints in english on test environments, it was the reason for CI tests failing and don't finding the login button.

 We've change the Capybara language to english for the moment but we think we should locate why is the configuration on tests put in english if TimeOverflow is mostly used in spanish, and do a better testing.

### PR DESCRIPTION
**Pull request by markets**
Closes #373 
Closes https://github.com/coopdevs/timeoverflow-provisioning/issues/24
Closes #544
Closes #548 

**EXTRA:**
- update more dev deps: rubocop, byebug, fabrication, capybara, webdrivers
- fixing CI (feature specs)

**INFRA CHANGES & TESTING**
👀 https://github.com/coopdevs/timeoverflow/pull/401#issuecomment-471716708

### CODE REVIEW

**Comment by enricostano** [comment]
To test this we first need to solve https://github.com/coopdevs/timeoverflow-provisioning/pull/36

**Comment by markets** [comment]
Ok @enricostano thanks! I think this will also close #369

File: app/jobs/organization_notifier_job.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+class OrganizationNotifierJob < ActiveJob::Base
```
Comment (sauloperez) [inline app/jobs/organization_notifier_job.rb]: It'd be great to explain what it notifies about and who without having to read the code. I'd be much faster for readers to better understand what it does.

**Comment by enricostano** [comment]
Cool @markets , many thanks for improving this PR! :muscle: 

Just two questions:

1. What would be the best strategy to test it?
2. I guess we should plan some kind of pre-post deploy operations for production to remove the existent cron job, right?

**Comment by markets** [comment]
Hi @enricostano 

1) the code inside the Job class is just a copy/pasta from the old Service class (with `find_each` instead of `each`), so in terms of logic, it should work exactly as it works now. To ensure that, I think we can use the Sidekiq UI, as cron jobs are displayed there and they have an "Enqueue Now" option:

Captura de pantalla 2019-03-11 a la(s) 21 21 32

2) We should remove the crontab line generated by the `whenever` gem, this can be done by hand (`crontab -e`) or using an option provided by the gem: `whenever -c`

**Comment by sseerrggii** [comment]
:thinking: I think I will need som help to test that @enricostano also I think we have som troubles with push notifications, so one day we can sit together and test thinks looking at  Sidekiq :pray:

**Comment by enricostano** [comment]
Blocked by https://github.com/coopdevs/timeoverflow-provisioning/pull/36

**Comment by markets** [comment]
Just updated with current develop! I also "unblocked" this PR, as the main blocker https://github.com/coopdevs/timeoverflow-provisioning/pull/36 was already solved.

**Comment by sseerrggii** [comment]
Testing it on staging i get this error

FireShot Capture 377 -  STAGING  Sidekiq - staging timeoverflow org

**Comment by markets** [comment]
Yay! Travis is passing ✅ now! It's already failing in master, and not related to this branch, but I fixed it here in order to have green build for this branch.

@sseerrggii I also fixed the error you got last day. Thanks!

**Comment by sauloperez** [comment]
Thanks a ton @markets ! what a mess with the locale :see_no_evil: @mperezv this is relevant to you :point_up:

**Comment by sseerrggii** [comment]
Now Works fine, thanks @markets :green_heart:

**Comment by sauloperez** [comment]
Finally up and working in production! thanks a lot @markets :tada:  ! TO is now cron-free and relies on Sidekiq for all kinds of async stuff :clap:

**Pull request merged**